### PR TITLE
Add CPANSA-HTTP-Tiny-2023-01 for HTTP::Tiny

### PR DESCRIPTION
**Pull request by stigtsp**
CVE is requested from MITRE, no patch available yet.

### CODE REVIEW

File: cpansa/CPANSA-HTTP-Tiny.yml [hunk 1]
```diff
@@ -1,4 +1,26 @@
 ---
+- id: CPANSA-HTTP-Tiny-2023-01
```
Comment (briandfoy) [inline cpansa/CPANSA-HTTP-Tiny.yml:2]: change from 2023 to 2015, the year of the report.

**Review by briandfoy — CHANGES_REQUESTED** [review]

File: cpansa/CPANSA-HTTP-Tiny.yml (hunk 1 above)
Comment (stigtsp) [reply cpansa/CPANSA-HTTP-Tiny.yml:2]: The reason for setting 2015 here was that it was the first entry I could find where it was pointed out as a problem. It wasn't discussed as a vulnerability (afaik) until 2022-2023 though, so maybe the report date should be set to 2023 instead? I'm not sure.

Comment (stigtsp) [reply cpansa/CPANSA-HTTP-Tiny.yml:2]: I've updated the reporting_date to 2023-02-14 which was when nerdvana posted https://www.reddit.com/r/perl/comments/111tadi/psa_httptiny_disabled_ssl_verification_by_default/

Comment (briandfoy) [reply cpansa/CPANSA-HTTP-Tiny.yml:2]: It looks like this has a CVE now, so use that for the sequence number in the `id` field. I noticed that the report below this one has that problem too.

Comment (stigtsp) [reply cpansa/CPANSA-HTTP-Tiny.yml:2]: Would you like to use the CVE as the id instead?

**Review by briandfoy — APPROVED** [review]

**Pull request merged**